lib/sensors/stripe_sensor.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <cstdint>
+#include <optional>
+
+#include "io/gpio.hpp"
+#include <core/logger.hpp>
+#include <io/hardware_gpio.hpp>

```

Comment:
`<io/gpio.hpp>` and remove `<io/hardware_gpio.hpp>`, the interface should suffice